### PR TITLE
Tidy up multi-simulation doc

### PR DESCRIPTION
**Pull request by hamishwillee**
Minor tidy up to doc following #825.

@lamping7 I will merge this, but would be good if you cold sanity check I did not change the meaning.

Note there are just a couple of points inline I have made to explain layout changes.

### CODE REVIEW

File: en/simulation/multi-vehicle-simulation.md [hunk 1]
```diff
@@ -62,7 +60,7 @@ For each simulated vehicle, the following is required:
 The launch file `multi_uav_mavros_sitl.launch`does the following,
 
 * loads a world in gazebo,
-```
+  ```
```
Comment (hamishwillee) [inline en/simulation/multi-vehicle-simulation.md:63]: @lamping7
Note
1. Typically use three backticks on a new line for a block of code and single backticks for an inline code. With a block you can specify the programming language for syntax highlighting (I have not done this here).
2. The backticks and blocks have to be indented next to the text in the bullet - in this case 2 spaces.

File: en/simulation/multi-vehicle-simulation.md [hunk 2]
```diff
@@ -121,25 +119,38 @@ To add a third iris to this simulation there are two main components to consider
...
-1. The `vehicle` argument is used to set the `PX4_SIM_MODEL` environment variable, which is used by the default rCS (startup script) to find the corresponding startup settings file for the model. Within PX4 these startup files can be found in the `Firmware/ROMFS/px4fmu_common/init.d-posix/` directory. For example, here is the plane model's [startup script](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d-posix/1030_plane). For this to work, the PX4 node in the launch file is passed arguemnts that specify the rCS file (`etc/init.d/rcS`) and the location of the rootfs directory (`$(find px4)/ROMFS/px4fmu_common`). For simplicity, it is suggested that the startup file for the model be placed alongside PX4's in `Firmware/ROMFS/px4fmu_common/init.d-posix/`.
+To add a new vehicle, you need to make sure the model can be found (in order to spawn it in Gazebo), and PX4 needs to have an appropriate corresponding startup script.
+
+1. You can choose to do either of:
+   * modify the **single_vehicle_spawn_sdf.launch** file to point to the location of your model by changing the line below to point to your model:
```
Comment (hamishwillee) [inline en/simulation/multi-vehicle-simulation.md:143]: I tend to use bold for filenames, italic for tool names, and code for code or command line invocations.

**Comment by lamping7** [comment]
LGTM

**Comment by hamishwillee** [comment]
Thanks for review!

**Pull request merged**